### ISSUE TITLE
Deepscan ?

### ISSUE THREAD

**Issue by evanlegamer** [DrakenWan/Yale3#4]
Hi, what is deepscan used for ?

I'm interested to work on this repo to improve it. Seems an interesting extension.

**Comment by DrakenWan** [comment]
A section on a linkedin page may have more entries in a list (say you had 12 past experiences in experience section ) then linkedin will only show a few entries from that list and others will be shown a new page if you click on show more. So to accomodate for this, I had to add an option and named it 'deepscan' (for some reason). All it does is accomodates for the extraction tool to extract all these entries when you go to the page of a particular section.

Here is a demonstration.

![Picture1](https://user-images.githubusercontent.com/13289205/200945262-10c15efd-865f-4475-9b0c-cd470d2b6da9.png)

In the picture above, Licenses and Certifications section has more than 3 entries. So LinkedIn only shows first 3 entries and lets you view other entries in a new page. So If I click on show more. It takes me to a new page with all entries from Licenses and Certifications. So I check the deepscan box and if I click on 'Extract certifications' button. It will give me all entries for licenses and certifications.

![Picture2](https://user-images.githubusercontent.com/13289205/200945760-e3199364-4081-4fba-b0b4-e5c464c68681.png)



I had to add this feature since linkedin has made changes that won't work with old code and this is the only possible way to accomodate for this latest change. It happened back in January 2022.
